content/Proposals/Accepted/202106-handbook.md

```diff
@@ -0,0 +1,133 @@
+# 2021-06: Handbook
+
+* **Owners:**:
+  * [`@bwplotka`](https://github.com/bwplotka)
+
+* **Other docs:**
+  * https://works.hashicorp.com/articles/writing-practices-and-culture
+  * https://about.gitlab.com/handbook/handbook-usage/#wiki-handbooks-dont-scale
+  * https://gitlab.com/gitlab-com/gl-infra/infrastructure/-/issues/5143
+  * [Observability Team Process (Internal)](https://docs.google.com/document/d/1eojXStPdq1hYwv36pjE-vKR1q3dlBbpIx5w_L_v2gNo/edit)
+  * [Monitoring Team Processes (Internal)](https://drive.google.com/drive/folders/1yU5DYhpBLzmp3q9SIhudRMZ1rsmQ_sR2)
+
+> TL;DR: I would like to propose to put all public documentation pieces related to the Observability Group (and not tied to a specific project) in the public GitHub repository called [`handbook`](https://github.com/rhobs/handbook). I propose to review all documents with a similar flow as code and put documents in the form of markdown files that can be read from both GitHub UI and automatically served on hhttps://rhobs-handbook.netlify.app/ website.
```

Comment:
This makes complete sense. I would just make it more explicit that developer docs (things like how to build, test and run a project) should be kept in the repository of the project itself.

Reply:
Yes, it's part of the flow below, but let me add explicitly here too.